data.sql

```diff
@@ -3,4 +3,36 @@ VALUES
 (1, 'Agumon', '2020-02-03', 0, true, 10.23),
 (2, 'Gabumon', '2018-11-15', 2, true, 8),
 (3, 'Pikachu', '2021-01-07', 1, false, 15.04),
-(4, 'Devimon', '2017-05-12', 5, true, 11);
\ No newline at end of file
+(4, 'Devimon', '2017-05-12', 5, true, 11),
+(5, 'Charmander', '2020-02-08', 0, false, -11),
+(6, 'Plantmon', '2022-11-15', 2, true, -5.7),
+(7, 'Squirtle', '1993-04-02', 3, false, -12.13),
+(8, 'Angemon', '2005-06-12', 1, true, -45),
+(9, 'Boarmon', '2005-06-07', 7, true, 20.4),
+(10, 'Blossom','1998-10-13', 3, true, 17);
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+ROLLBACK;
+BEGIN;
+UPDATE animals
+SET species = 'digimon'
+WHERE animal_name LIKE '%mon';
+UPDATE animals
+SET species = 'pokemon'
+WHERE species is NULL;
+COMMIT;
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+BEGIN;
+DELETE FROM animals
+WHERE date_of_birth > '01-01-2022';
+SAVEPOINT ROWS9;
+UPDATE animals
+SET weight_kg = weight_kg * -1;
+ROLLBACK TO ROWS9;
+UPDATE animals
+SET weight_kg = weight_kg * -1
+WHERE weight_kg < 0;
+COMMIT;
```

Comment:
- [ ] <b>REQUIRED:</b> All of those query commands from line 13 to line 38 are supposed to be put in `query.sql` file and make sure that you use `SELECT` command after each update to verify that the change was made. something like this:

```sql
   BEGIN;
   UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon'; 
   UPDATE animals SET species = 'pokemon' WHERE species IS NULL; 

   SELECT species from animals; -- verify that change was made
   COMMIT;

Reply:
I was thinking to do it in the data.sql file because is not specified but I already updated the files, thank you! 🚀